### PR TITLE
Update README.md

### LINKED ISSUE #105: PyPI main.py relocation breaks the documented way of running CLI help

### PR DESCRIPTION
**Pull request by mservidio**
Fixes #105

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,22 +179,6 @@ pip install -r requirements.txt
...
-```
-
-## Creating a shortcut
-
-If you prefer to make the transcoder available globally on your machine, execute the scripts below. This script will update the permissions on the txcode symlink to grant execute permissions, and add the txcode symlink to your system PATH variable.
-
-Note if using macOS with interactive shells, you will need to change '\~/.bash_profile' to '\~/.bash_rc'.
-
-```
-cd market-data-transcoder/
```
Comment (mservidio) [inline README.md:192]: @salsferrazza I think at this point we can remove bin/txcode? The PyPI setup is referencing the main.py file directly for the alias.

Comment (salsferrazza) [reply README.md:192]: I agree, as long as `txcode` is addressable after installation having it in `/bin` is superfluous.

Comment (mservidio) [reply README.md:192]: bin is removed, and updated the examples documentation to use 'txcode'

**Comment by mservidio** [comment]
@salsferrazza @t8dogg Could you please review?

**Review by t8dogg — APPROVED** [review]
Thanks for these changes Mark, I was grappling with exactly this setup step earlier this week!

**Pull request merged**